iommu_ref_model/libiommu/src/iommu_reg.c

```diff
@@ -818,7 +819,8 @@ reset_iommu(uint8_t num_hpm, uint8_t hpmctr_bits, uint16_t eventID_mask,
             uint8_t num_vec_bits, uint8_t reset_iommu_mode, 
             uint8_t max_iommu_mode, uint32_t max_devid_mask,
             uint8_t gxl_writeable, uint8_t fctl_be_writeable,
-            capabilities_t capabilities, fctl_t fctl) {
+            uint8_t fill_ats_trans_in_ioatc, capabilities_t capabilities,
```

Comment:
This option fill_ats_trans_in_ioatc is only an implementation option ?
It is not SW discoverable nor configurable.

Reply:
Yes, Its an implementation option. When dealing with plug-in devices its not useful. However for integrated devices this may be useful.